### PR TITLE
Erase all unattended-upgrades options first

### LINKED ISSUE #94: apt options are not overridden but merged
Hello,
thank you for this wonderful Ansible role, especially for the approach of overriding the unattended-upgrades config file instead of replacing it.
Unfortunately, this doesn't work as intended because config options for apt are not overridden but merged. For details see the solution to [this old issue](https://serverfault.com/questions/924538/unattended-upgrades-override-allowed-origins).

Therefore options should be cleared before the override is applied, e.g.
`#clear Unattended-Upgrade::Origins-Pattern;`
should precede the line 
`  Unattended-Upgrade::Origins-Pattern {`

Kind regards

### PR DESCRIPTION
**Pull request by Normo**
This MR adds a `#clear` section at the top of the configuration template in order to delete first all relevant unattended-upgrade options that may have been set in other configuration files.

Fixes #94

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+# List of unattended-upgrades options that need to be erased before set by this role
+# See https://github.com/hifis-net/ansible-role-unattended-upgrades/issues/94
+__clear_options:
```
Comment (tobiashuste) [inline vars/main.yml:4]: How did you come up with this list of options? Is there some link with the available options somewhere?

Also, how can we make sure to not miss the addition of newly added variables?

**Review by tobiashuste — COMMENTED** [review]
Thanks, @Normo. This looks very reasonable and like a good approach :+1:

I only had one question :)

File: vars/main.yml (hunk 1 above)
Comment (Normo) [reply vars/main.yml:4]: These were my sources:
* [https://github.com/mvo5/unattended-upgrades#supported-options-reference](https://github.com/mvo5/unattended-upgrades#supported-options-reference)
* [`50unattended-upgrades.Ubuntu`](https://github.com/mvo5/unattended-upgrades/blob/master/data/50unattended-upgrades.Ubuntu)
* [`50unattended-upgrades.Debian`](https://github.com/mvo5/unattended-upgrades/blob/master/data/50unattended-upgrades.Debian)

> Also, how can we make sure to not miss the addition of newly added variables?

Well, check the new unattended-upgrades releases regularly.

Comment (tobiashuste) [reply vars/main.yml:4]: Thanks :+1:

**Review by tobiashuste — APPROVED** [review]
LGTM :shipit:

**Pull request merged**